platform/working-with-vsp/orientation/request-access-to-tools.md

```diff
@@ -1,91 +1,94 @@
     * Working on: ```The project your team is working on```, e.g., "526 ancillary forms"
 
-1. Developers, complete the [additional VSP orientation steps](#additional-orientation-steps-for-developers) below to access the code repositories and tools you'll need for development and deployment.
-
-## Zenhub
-
-1. Zenhub is used by the VSP team for issue tracking and management.
+### Getting Repo Access 
+1. Once you are in the VA GitHub Organization, attend Orientation with the Product Support team.
 
-2. Additional details and orientation on Zenhub can be found [in this document](./zenhub.md).
+1. After Orientation, VSP will add you to the `vets.gov-write` team, which will provide you access to the repositories needed for working on VA.gov.
+    * If you still don't have this access, post a request in [#vfs-platform-support](https://dsva.slack.com/channels/vfs-platform-support).
 
-## Additional orientation steps for developers
+## ZenHub
 
-The internal tools available include Grafana, Sentry, Prometheus, and Jenkins. These tools are hosted internally and
-available for developers via proxy access. We use `ssh` and the Chrome plugin SwitchyOmega to route web requests from
-your browser to the SOCKS5 proxy for these private domains. For this we require the use of an `ssh` key pair to secure access.
+1. ZenHub is used by the VSP team for issue tracking and management.
 
-In addition to internal tools our infrastructure is hosted in and makes use of services in AWS GovCloud. This account is shared amongst many different teams; changes made to resources in this account will affect many services. You will need access to AWS for troubleshooting, log file review, and to apply changes to the running instances and services. 
+2. Additional details and orientation on ZenHub can be found [in this document](./zenhub.md).
 
-For either SOCKS5 proxy and/or AWS Console access please create a "Environment Access Request" issue in Github. All SOCKS5 proxy access requests are reviewed by VSP Operations. 
+## Additional access for developers
 
-In order to be granted AWS Console access, your PIV background (eQIP adjudication) check needs to be underway(for VSP Operations folks, this process is expedited). Please mark yes in the AWS Console access and provide a short description of why you need access. All AWS Console access requests will be reviewed by DEPO. **If you do not need access to AWS please mark "No" in the Grant AWS Console Access section**. Please ensure  all VA onboarding paperwork is submitted to your company/COR.
+Several internal tools for development are only accessible within the VA network. Access to developers outside the VA network is made available via a SOCKS proxy. For proxy access, we require the use of an SSH key pair for security.
 
-We will be walking through how to request access in the steps outlined below.
+Follow the below steps to set up your SSH key pair, request access, and set up the SOCKS proxy:
```

Comment:
Consider removing this sentence and using subheadings for Set up SSH key pair, Request access, and Set up the SOCKS proxy (instead of having one long series of steps).